### DISCUSSION TITLE
Quirks mode

### DISCUSSION THREAD

**Discussion by frederickz** [jpanther/congo#50] [Q&A]
I restarted building my website from the very beginning. Now I just finished "Install using Hugo" section, but when I enter `localhost:1313` in Chrome, it shows up a blank page. In console, it warns
```
WARN 2021/12/13 21:12:21 found no layout file for "HTML" for kind "home": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
```
In the inspect panel in Chrome, it warns:
```
Page layout may be unexpected due to Quirks Mode
```
I just updated Golang, Hugo, Congo, and Chrome. They are all up to dates. I also follows the Hugo new site with their theme provided (https://gohugo.io/getting-started/quick-start/), which is fine.

**Comment by frederickz** [comment]
It seems a problem with Hugo and `hugo mod`. Before the question, I was using `hugo mod`. Now I turned to `git submodule` and the problem fixed.

**Reply by jpanther** [reply]
Interesting. I just followed the installation instructions using a Hugo module and this worked fine for me (I'm testing using Hugo v0.90.1). Glad you got it working with git in any case.

**Reply by frederickz** [reply]
Thanks James! Not sure what happened on my side. But since I am about to override the template a little bit, it is better to have a submodule in hand.

**Reply by adam-sroka** [reply]
For those who found this discussion while trying to fix the `WARN` message, you might also want to check out #382 😊.
